i18n-helpers/CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 This lists the most important changes between releases of mdbook-i18n-helpers.
 
-## Version 0.3.4 (2024-05-26)
+## Version 0.3.5 (2024-07-23)
+
+- [#208] : Trim dependencies by @klensy
+- [#209], [#210], [#213], [#214]: Utility for generating diff for PO files by @qwandor
+- [#212]: Advance POT-Creation-Date by 1 minute by @carreter
+- [#215]: Fix granularity in sub-chapters by @carreter
+
+## Version 0.3.4 (2024-06-14)
```

Comment:
Changed the date for older release because it was wrong https://github.com/google/mdbook-i18n-helpers/pull/204